resources/authgear/templates/en/web/__translation_message.html

```diff
@@ -0,0 +1,9 @@
+{{ define "__translation_message.html" }}
+<span
+  class="translated-message"
```

Comment:
This could be a class with style
```
display: contents;
```
To ensure it won't affect any layout.

Reply:
👍🏻